src/sampleData/roomsData.js

```diff
@@ -0,0 +1,35 @@
+const rooms = [{
+    "number": 1,
+    "roomType": "residential suite",
+    "bidet": true,
+    "bedSize": "queen",
+    "numBeds": 1,
+    "costPerNight": 358.4
+  },
+  {
+    "number": 2,
+    "roomType": "suite",
+    "bidet": false,
+    "bedSize": "full",
+    "numBeds": 2,
+    "costPerNight": 477.38
+  },
+  {
+    "number": 3,
+    "roomType": "single room",
+    "bidet": false,
+    "bedSize": "king",
+    "numBeds": 1,
+    "costPerNight": 491.14
+  },
+  {
+    "number": 4,
+    "roomType": "residential suite",
+    "bidet": true,
+    "bedSize": "queen",
+    "numBeds": 1,
+    "costPerNight": 358.4
+  }
+];
+
```

Comment:
I suggest adding at least one more room to your sample data since you're not currently accounting for roomType 'junior suite'.